Incorrect return type hint causes airnow HA component to fail CI testing

`observation.Observations.latLong()` is incorrectly hinted to return `None` instead of `list`.  This triggered the following errors from mypy during homeassistant's [CI test run](https://github.com/home-assistant/core/actions/runs/15834487385/job/44639620195?pr=147388):

```
Error: homeassistant/components/airnow/coordinator.py:76: error: Function does not return a value (it only ever returns None)  [func-returns-value]
Error: homeassistant/components/airnow/coordinator.py:88: error: Statement is unreachable  [unreachable]
Error: homeassistant/components/airnow/config_flow.py:43: error: Function does not return a value (it only ever returns None)  [func-returns-value]
Error: homeassistant/components/airnow/config_flow.py:56: error: Statement is unreachable  [unreachable]
```

I'll create a PR with a fix momentarily.

--- a/pyairnow/api.py
+++ b/pyairnow/api.py
@@ -1,6 +1,6 @@
 '''Client to interact with AirNow Air Quality API'''
 from json.decoder import JSONDecodeError
-from typing import Optional
+from typing import Optional, cast
 
 from aiohttp import ClientSession, ClientTimeout
 
@@ -20,7 +20,7 @@ class WebServiceAPI:
     ) -> None:
         '''Initialize with Client Session and API Key'''
         self._api_key: Optional[str] = api_key
-        self._session: ClientSession = session
+        self._session: Optional[ClientSession] = session
 
         self.forecast = Forecast(self._get)
         self.observations = Observations(self._get)
@@ -37,7 +37,7 @@ class WebServiceAPI:
 
         session: ClientSession
         if use_running_session:
-            session = self._session
+            session = cast(ClientSession, self._session)
         else:
             session = ClientSession(
                 timeout=ClientTimeout(total=API_DEFAULT_TIMEOUT)

--- a/pyairnow/conv.py
+++ b/pyairnow/conv.py
@@ -1,6 +1,19 @@
 '''Convert AQI to and from Pollutant Concentrations'''
+from typing import TypedDict
 
-EPA_TABLE = [
+
+class ConcBreakpointPair(TypedDict):
+    bL: float
+    bH: float
+
+
+class EPATableRow(TypedDict):
+    iL: int
+    iH: int
+    breakpoints: dict[str, ConcBreakpointPair]
+
+
+EPA_TABLE: list[EPATableRow] = [
     {
         'iL': 0,
         'iH': 50,
@@ -86,7 +99,7 @@ EPA_ROUND_FNS = {
 }
 
 
-def aqi_to_concentration(aqi, pollutant):
+def aqi_to_concentration(aqi: int, pollutant: str) -> float:
     '''Convert AQI (0-500) to Pollutant Concentration'''
     if aqi < 0:
         raise ValueError('AQI must be greater than 0')
@@ -107,7 +120,7 @@ def aqi_to_concentration(aqi, pollutant):
     raise RuntimeError('No matching row found in EPA_TABLE')
 
 
-def concentration_to_aqi(conc, pollutant):
+def concentration_to_aqi(conc: float, pollutant: str) -> int:
     '''Convert Pollutant Concentration to AQI'''
     if conc < 0:
         raise ValueError('Concentration must be positive')

--- a/pyairnow/observation.py
+++ b/pyairnow/observation.py
@@ -32,7 +32,7 @@ class Observations:
         longitude: Optional[Union[float, str]] = None,
         *,
         distance: Optional[int] = None,
-    ) -> None:
+    ) -> list:
         '''Request current observation for latitude/longitude'''
         params: dict = dict(
             latitude=str(latitude),

